operations.adoc

```diff
@@ -70,18 +71,33 @@ For each final specification:
 A specification document that is marked "Final" cannot be made generally available until after engaging in a successful release review (with corresponding super-majority approval from the specification committee).
 A release review will have validated that the specification project has:
 
-1. Produced a staging release via OSSRH staging repository for the api, javadoc. The specification would be staged on a github release marked as pre-release.
-2. Attached the candidate final specification to the release in English
-3. The candidate final EFTL licensed TCK archive should be uploaded to the project directory under download.eclipse.org.
-The github release should have a link to the candidate EFTL licensed artifact.
-There is no need to produce a final EPL version?
-Question on issues with using mirrored links.
-4. Attached the EFTL TCK summary results for each implementation included with this release review.
-5. Create two https://help.github.com/en/articles/about-pull-requests#draft-pull-requests[draft PRs] request against the
+1. Produced a staging release via OSSRH staging repository for the api, javadoc.
+** Javadocs should include the https://raw.githubusercontent.com/eclipse-ee4j/jakartaee-api/master/licenses/EFSL.html[Eclipse Foundation Specification License].
+** Version in pom.xml should be increased by comparing to the previously released API artifact.
+** Existing "@version ..." JavaDoc tags should be updated to match, or removed.
+1. The candidate final https://www.eclipse.org/legal/tck.php[EFTL] licensed TCK archive should be uploaded to the project directory under http://download.eclipse.org/ee4j, e.g., http://download.eclipse.org/ee4j/bean-validation/beanvalidation-tck-dist-2.0.5.zip .
+1. Generate standalone TCK results or platform TCK result as apporiate for the spec project. 
+** Information on running TCKs in the Jakarta CI infrastructure can be found https://wiki.eclipse.org/TCK:Build_From_Jakarta_EE_TCK_Repo_And_Run#Jenkins_Pipelines[here].
```

Comment:
I personally don't like the "here" hotlinks.  They are too easy to miss.  I'd prefer something like "can be found in the 'url'[Jakarta Jenkins documentation]".  But, this is just a nit...